### PR TITLE
refactor(frontend): ignore stmts  START TRANSACTION and ABORT

### PR DESCRIPTION
**Pull request by likzn**
## What's changed and what's your intention?
As issue #2595 say. We return them directly before the investigation of the behavior of pg . 

**Main changes:**
1. Support `parser` to parse `ABORT`
2. `src/frontend/src/handler/mod.rs#handler` will return empty response when meet `START TRANSACTION` and `ABORT`

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

## Refer to a related PR or issue link (optional)
#2595

### CODE REVIEW

File: src/utils/pgwire/src/pg_response.rs [hunk 1]
```diff
@@ -46,6 +46,8 @@ pub enum StatementType {
     SET_OPTION,
     SHOW_PARAMETERS,
     SHOW_COMMAND,
+    START_TRANSITION,
```
Comment (neverchanje) [inline src/utils/pgwire/src/pg_response.rs:49]: START_TRANSACTION

File: src/frontend/src/handler/mod.rs [hunk 2]
```diff
@@ -131,6 +132,8 @@ pub(super) async fn handle(session: Arc<SessionImpl>, stmt: Statement) -> Result
             }
             create_index::handle_create_index(context, name, table_name, columns).await
         }
+        Statement::StartTransaction { .. } => Ok(PgResponse::empty_result(START_TRANSITION)),
```
Comment (neverchanje) [inline src/frontend/src/handler/mod.rs:135]: Pls add a Todo comment here to note it's intended and not a final implementation

Comment (likzn) [reply src/frontend/src/handler/mod.rs]: All Done

File: src/frontend/src/handler/mod.rs [hunk 3]
```diff
@@ -131,6 +132,12 @@ pub(super) async fn handle(session: Arc<SessionImpl>, stmt: Statement) -> Result
             }
             create_index::handle_create_index(context, name, table_name, columns).await
         }
+        // Ignore `StartTransaction` and `Abort` temporarily.Its not final implementation.
+        // 1. Fully support transaction is too hard and gives few benefits to us.
+        // 2. Some client e.g. psycopg2 will use this statement.
+        // TODO: Track issues #2595 #2541
+        Statement::StartTransaction { .. } => Ok(PgResponse::empty_result(START_TRANSACTION)),
```
Comment (neverchanje) [inline src/frontend/src/handler/mod.rs:139]: lgtm, but it will be better to use `empty_result_with_notice` and give users a warning.

Comment (likzn) [reply src/frontend/src/handler/mod.rs]: Updated~  It will return below
```
Ignored temporarily.See detail in issue#2541
START_TRANSACTION
```

**Comment by likzn** [comment]
@neverchanje I find CI failed again in the same problem.So i update branch, but i find i cannot trigger CI. Maybe need you to trigger.

**Pull request merged**